packages/aws-cdk/README.md

```diff
@@ -579,6 +613,11 @@ To import an existing resource to a CDK stack, follow the following steps:
 5. When `cdk import` reports success, the resource is managed by CDK. Any subsequent
    changes in the construct configuration will be reflected on the resource.
 
+NOTE: You can also import existing resources by `--import-existing-resources` option of `cdk deploy` command.
+This parameter only works for resources that you can set custom physical names,
+such as S3 bucket, DynamoDB table, etc.
```

Comment:
```suggestion
such as S3 bucket, DynamoDB table, etc...
```

Reply:
Love that you're pushing this change and really appreciate the contribution. I've wanted this functionality for a while so it's great to see it being implemented. Just have some doc changes and it's g2g imo